metadata_tools.py

```diff
@@ -41,10 +41,15 @@ def read_exif_tags(self):
     def write_exif_tags(self, exif_dict):
         """Writes all exif tags to an image with a single call to ExifTool"""
         self.logger.info(f"Processing EXIF data for: {self.path}")
+
```

Comment:
if any value return by dictionary is not valid , will raise error and return booleans of which terms are not valid.